py-deploy/tldeploy/identity.py

```diff
@@ -0,0 +1,141 @@
+from eth_keys.datatypes import PrivateKey
+from tldeploy.signing import solidity_keccak, sign_msg_hash
+
+
+class MetaTransaction():
+
+    def __init__(
+        self,
+        *,
+        from_: str = None,
+        to: str,
+        value: int = 0,
+        data: bytes = bytes(),
+        nonce: int = None,
+        extra_hash: bytes = bytes(),
+        signature: bytes = None,
+    ):
+        self.from_ = from_
+        self.to = to
+        self.value = value
+        self.data = data
+        self.nonce = nonce
+        self.extra_hash = extra_hash
+        self.signature = signature
+
+    @classmethod
+    def from_function_call(
+        cls,
+        to: str,
+        function_call,
+        *,
+        nonce: int = None,
+        from_: int = None,
```

Comment:
I don't get this function, but should `from_` not be before `to` and also be a `str`